### DISCUSSION TITLE
How to exactle build the iso that is downloadable from the homepage

### DISCUSSION THREAD

**Discussion by CarstenMaul** [ipxe/ipxe#950] [Q&A] [answered]
Hi all,

how can I build the ISO file that is downloadable from the homepage by myself?
https://boot.ipxe.org/ipxe.iso

I want to learn how to exactly replicate this iso with all settings.

I downloaded the repository and i am able to build isos. But the ISOs that I get are not exactly like the one that the download provides.

Thanks.

**Comment by NiKiZe** [comment, marked as answer]
Hopefully this comment explains this https://github.com/ipxe/ipxe/discussions/614#discussioncomment-2363937
